### ISSUE TITLE
Wrangler now includes miniflare, but DurableObjects are still not solved.

### ISSUE THREAD

**Issue by gerhardcit** [eltigerchino/sveltekit-cloudflare-template#1]
The latest create-cloudflare cli create this hooks file when you select sveltekit.

`hooks.ts`
```
let platform: App.Platform;

if (dev) {
	const { getPlatformProxy } = await import('wrangler');
	platform = await getPlatformProxy();
}

export const handle = async ({ event, resolve }) => {
	if (platform) {
		event.platform = {
			...event.platform,
			...platform
		};
	}

	return resolve(event);
};
```

Have you tried using the new `wrangler` approach. I would love to know how DurableObjects are solved in dev, or even
`npx wrangler pages dev .svelte-kit/cloudflare`

**Comment by eltigerchino** [comment]
Hi thanks for bringing this up! I've forgotten to update this template since they've introduced this new API.

It seems that durable objects are supported! https://developers.cloudflare.com/workers/wrangler/api/#supported-bindings While I have no experience with durable objects myself, I'll try and figure something out when I have the time. It must be configured in the wrangler.toml file.

**Comment by gerhardcit** [comment]
Yes, DurableObjects are not really support in Pages though. You can get a binding, but the Class code is somehow not handled well in the bind process.

**Comment by eltigerchino** [comment]
Is that SvelteKit issue or is it a general issue with Pages? There are steps shown in the cloudflare pages docs so I’ll try to follow them and see if Kit can do anything more to match it https://developers.cloudflare.com/pages/functions/bindings/#durable-object-namespaces

**Comment by eltigerchino** [comment]
Important note on Durable Objects and Cloudflare Pages compatibility
https://github.com/cloudflare/cloudflare-docs/issues/13062#issuecomment-1953919015

Haven't tried the below but someone got it working (using a separate worker and CF pages app)
https://github.com/cloudflare/cloudflare-docs/issues/13062#issuecomment-1953341484
